cms/tests/test_site.py

```diff
@@ -63,7 +64,12 @@ def test_site_preview(self):
             # simulate user clicks on preview icon
             response = self.client.get(admin_reverse('cms_page_preview_page', args=[page.pk, 'de']))
             self.assertEqual(response.status_code, 302)
-            self.assertEqual(response._headers['location'][1], 'http://sample2.com{}&language=de'.format(page_edit_url_on))
+            if DJANGO_2_2 or DJANGO_3_0 or DJANGO_3_1:
```

Comment:
This is strange to do, normally you would have a separate test for the versions and use SkipIf(DJANGO_2_2) decorated on the function. I guess this will be ok to leave.